sysbox-in-docker/sysbox

```diff
@@ -144,42 +144,26 @@ function kernel_config_setup() {
   fi
 }
 
-#
 # Obtain the MTU value to be configured for the docker interface within the test
 # container. This value shall be the lowest among these ...
 #
-#  * The 'physical' default-gateway interface at host level (L0).
-#  * The 'logical' default-gateway interface connecting the test-container (L1) with the host.
+#  * The privileged-container's egress interface (host facing iface).
 #  * The 'default' mtu value (1500 bytes) supported by most NICs.
-#
 function docker_iface_mtu() {
 
-  local egress_iface
-  local l0_egress_mtu
-  local l1_egress_mtu
-
   # Identify default egress iface.
   local egress_iface=$(ip route show | awk '/default via/ {print $5}')
   if [ -z "${egress_iface}" ]; then
 	  return
   fi
 
   # Obtain mtu value associated to the test-container's egress interface.
```

Comment:
nit: "test-container" -> "privileged-container" ?

Reply:
Missed that one, thanks.